uWeather/app/src/main/java/dit257/mandalore/uweather/Probability.kt

```diff
@@ -1,36 +1,36 @@
 package dit257.mandalore.uweather
 
+import java.lang.Math.*
 import java.util.*
+import kotlin.math.abs
 
 class Probability {
 
     //Calculate probability based on difference from median
```

Comment:
please make a more clear description that it takes an array of temperatures from different weather services to calculate a probable (is aggregated the right word?) forecast.

Reply:
Will update the comments :)